packages/labs/analyzer/src/lib/analyzer.ts

```diff
@@ -42,14 +30,15 @@ export class Analyzer {
     // TODO(kschaaf): Consider moving the package.json and tsconfig.json
     // to analyzePackage() or move it to an async factory function that
     // passes these to the constructor as arguments.
+    let packageJson;
     try {
-      this.packageJson = JSON.parse(
+      packageJson = JSON.parse(
         fs.readFileSync(path.join(packageRoot, 'package.json'), 'utf8')
       );
     } catch (e) {
       throw new Error(`package.json not found in ${packageRoot}`);
```

Comment:
btw, there's two potential errors here: no file and malformed JSON.

Reply:
Fixed